jsonschema2pojo-integration-tests/src/test/java/org/jsonschema2pojo/integration/DefaultIT.java

```diff
@@ -165,15 +164,6 @@ public void utcmillisecPropertyHasCorrectDefaultValue() throws NoSuchMethodExcep
     }
 
     @Test
-    public void uriPropertyHasCorrectDefaultValue() throws Exception {
```

Comment:
I think there is a git merge error here. I don't believe you intended to delete this test.